src/providers/UserProvider.js

```diff
@@ -0,0 +1,44 @@
+import crypto from 'crypto';
+import * as BaseProvider from './BaseProvider';
+
+/* Provider for User Registration */
+const create = (model, body) => {
+    return new Promise((resolve, reject) => {
+        if (body.user_name == '' && body.email == '' && body.password == '' && body.confirm_password == '') {
```

Comment:
use this library for all validation 

https://github.com/ctavan/express-validator